android/build.gradle

```diff
@@ -2,30 +2,30 @@ group 'io.github.zeshuaro.google_api_headers'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.5.30'
```

Comment:
The latest Flutter generates and sets this to `1.6.10`, can we use this version instead?

Reply:
Good catch, fixed.